apps/site/pages/docs/how-to-preview-headless-cms-changes-in-development-mode.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: 'How to preview Headless CMS changes in development mode'
+sidebar_label: 'How to preview Headless CMS changes in development mode'
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+# How to preview Headless CMS changes in development mode
+
+To make local tests, you can use the `dev mode` of the Headless CMS
+to see a preview of your changes.
+
+<Callout type="warning" emoji="⚠️">
+  While this guide is applicable to other frontend projects 
+  within the Headless CMS, the use case for this guide is for FastStore projects.
+</Callout>
+
+ 
+
+1. After making changes to your FastStore project directory and syncing them with the Headless CMS, 
+access the VTEX Admin and navigate to **Storefront > Headless CMS**.
+2. Ensure you are in your FastStore project and open the page where changes were made.
+3. Open the **Developer Tool** or right-click on the page and choose **Inspect**.
+4. In the Developer Tool terminal and run the following command:
+
+```bash copy
+localStorage.setItem('cmsDevMode', 1)
+```
+
+5. Refresh the page and you should see the section **Development Mode**.
+
+![development-mode](https://vtexhelp.vtexassets.com/assets/docs/src/development-mode___5cd79e364aea0785e98ce6bb379f2acc.png)
+
+6. Click on `Preview`. A new tab will open; copy the link from this tab containing the
+`contentType`, `documentId` and `versionId`. The link should be similar to the
+following:
+`https://faststore-poc.vtex.com/?contentType=page&documentId=bd9e01d6-534e-4e54-aa76-f864dcbad024&versionId=405118e9-7509-11ed-83ab-0eeee363631w`
+
+7. Now access the preview link locally and change the parameters accordingly:
+
+```copy
+localhost:3000/api/preview/?contentType=<contentType>&documentId=<DOCUMENT_ID>&versionId=<VERSION_ID>
+```
+
+<Callout type="warning" emoji="⚠️">
+  Ensure you replace `<contentType>`, `<DOCUMENT_ID>`, and `<VERSION_ID>` with the actual values from your preview link.
+</Callout>
+
```

Comment:
Given the previous example, the local preview link should be the following:

`localhost:3000/api/preview/?contentType= page&documentId=bd9e01d6-534e-4e54-aa76-f864dcbad024&versionId=405118e9-7509-11ed-83ab-0eeee363631w`